inference-engine/tests/functional/inference_engine/ir_serialization/tensor_names.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <file_utils.h>
+#include <ie_api.h>
+#include <ie_iextension.h>
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include "ie_core.hpp"
+#include "ngraph/ngraph.hpp"
+#include "transformations/serialize.hpp"
+#include <ngraph/opsets/opset6.hpp>
+
+class TensorNameSerializationTest : public ::testing::Test {
+protected:
+    std::string test_name =
+        ::testing::UnitTest::GetInstance()->current_test_info()->name();
```

Comment:
nit: you can use ie_test_utils helpers here to get unique file name ([example](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/inference_engine/ir_serialization/serialize.cpp#L31)) it is needed when running multiple instances of the same test via gtest-parallel; probably not the issue right now but I already fixed one bug in CI related to this kind of issue, so I'm paying attention

> const std::string test_name =  [GetTestName()](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/ie_test_utils/common_test_utils/test_common.cpp#L78) + "_" + [GetTimestamp()](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/ie_test_utils/common_test_utils/test_common.cpp#L71)

Reply:
Hmmm, in this case we will generate different test name for each run...

Reply:
Yeah i see it now, the name is misleading ;) The intention is to generate unique file names per each run.

Reply:
Oh, got it. Fixed